http/action/src/test/java/io/knotx/databridge/http/action/HttpActionTest.java

```diff
@@ -85,8 +88,9 @@
       "  \"url\": \"http://knotx.io\",\n" +
       "  \"label\": \"Product\"\n" +
       "}";
-  private static final Fragment FRAGMENT = new Fragment("type", new JsonObject(), "expectedBody");
+  //  private static final Fragment FRAGMENT = new Fragment("type", new JsonObject(), "expectedBody");
   private WireMockServer wireMockServer;
+  private ActionLogLevel actionLogLevel = ActionLogLevel.INFO;
```

Comment:
Do we need this variable?

Reply:
Yes, we are using it to instantiate HttpAction, in a normal way it would be done using HttpActionFactory, therefore in all tests we are passing this argument.

Reply:
fixed